src/main/kotlin/nl/tudelft/ewi/se/ciselab/testgenie/settings/SettingsPluginComponent.kt

```diff
@@ -62,8 +72,18 @@ class SettingsPluginComponent {
         val width = panel?.visibleRect?.width
         val height = panel?.visibleRect?.height
 
+        // Simplify colorPicker
+        colorPicker.removeChooserPanel(colorPicker.chooserPanels.component1())
+        colorPicker.removeChooserPanel(colorPicker.chooserPanels.component2())
+        colorPicker.removeChooserPanel(colorPicker.chooserPanels.component2())
+        colorPicker.removeChooserPanel(colorPicker.chooserPanels.component2())
```

Comment:
Is this the only way to remove those panels? Is there to initialize the color picker without those components in the first place?

Reply:
From my research and experimentation, this is unfortunately the only method. This component is older than some dinosaurs.

Reply:
Ohh, I see.

Reply:
😳